src/Components/NavBar.js

```diff
@@ -2,10 +2,20 @@
 import "./NavBar.css";
 
 const NavBar = () => {
+  const [topics, setTopics] = useState([]);
+
+  useEffect(() => {
+    getTopics().then((topicsData) => {
+      setTopics(topicsData);
+    });
+  }, []);
 
   return (
     <div className="navBar-container">
-      <button id="navBar-button"> SEE ARTICLES BY TOPICS </button>
+      {topics.map((topic) => {
+        return <Link className="navBar-button" key={topic.slug}>{topic.slug} news</Link>
```

Comment:
So on this task, we would like you to render a list of articles filtered by the topic selected on a new page. For example a link to /articles/coding for all the going articles. Give me a shout if you need me to clarify this at all 🙂